File: CursoPOOUber/PHP/car.php
<?php
class Car 
{
    public id;
    public license;
    public driver;
    public passanger;

    public function __construc($license , $driver) {
        $this->license = $license;
        $this->driver = $driver;
    }

    public function printDataCar() {
        echo "license: $this->license, conductor: {$this->driver->name}, document: {$this->driver->document}"
    }
}

File: CursoPOOUber/Java/Car.java
class Car {
    Integer id;
    String license;
    Account drive;
    Integer passenger;

    public Car(String license, Account driver){
        this.license = license;
        this.drive = driver;
    }

    void printDataCar() {
        System.out.println("License: " + license + " Name Driver: " + drive.name);
    }
}
